### PR TITLE
Fix naming conflict with huggingface datasets

### PR DESCRIPTION
**Pull request by terafo**
lm-eval uses huggingface datasets for everything and name conflicts from that are annoying and unnecessary.

### CODE REVIEW

**Comment by terafo** [comment]
Ok, I haven't noticed comment edits in my original PR. tinydatasets is indeed quite long and similar enough to tinygrad to be confused. Would something like `nanodata`/`petitedata`/`babysets` be an OK name?

**Comment by mdatsev** [comment]
extra/datasets #1155

**Comment by g1y5x3** [comment]
Or just put it under extra/ like he suggested in discord

File: examples/mnist_gan.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from tinygrad.tensor import Tensor
 from tinygrad.helpers import getenv
 from tinygrad.nn import optim
-from datasets import fetch_mnist
+extra.datasets import fetch_mnist
```
Comment (jla524) [inline examples/mnist_gan.py:10]: from extra

**Pull request merged**